molecule/default/tasks/gitea_test.yml

```diff
@@ -109,7 +109,10 @@
 
 - name: Verify that Gitea failed to be created from the CR when appropriate
   assert:
-    that: namespace_modded_cr.resources[0].status is not defined == (scope == "namespace")
+    that: namespace_modded_success == namespace_modded_expected
+  vars:
+    namespace_modded_success: '{{ namespace_modded_cr.resources[0].status is defined }}'
+    namespace_modded_expected: '{{ scope == "cluster" }}'
```

Comment:
*Nice.*

Reply:
Yeah it makes it easier to read by a lot, but also improves reliability on newer ansible versions that didn't like the old way, so.....